code/modules/mob/living/blood.dm

```diff
@@ -283,7 +283,8 @@
 	if(QDELETED(B)) //Give it up
 		return
 	if(B.count < 10 )
-		B.color = blood_type.color
+		if(blood_type)
+			B.color = initial(blood_type.color)
```

Comment:
Why initial?

Reply:
because for some reason blood_id can return non created blood objects and in that cause i want to read the value of the initial type that it is, actually the fix for this is to create a new reagent type for this instead of initial